src/coreclr/jit/promotion.cpp

```diff
@@ -20,7 +20,7 @@ PhaseStatus Compiler::PhysicalPromotion()
         return PhaseStatus::MODIFIED_NOTHING;
     }
 
-    if (!compStressCompile(STRESS_PHYSICAL_PROMOTION, 25))
+    if ((JitConfig.JitEnablePhysicalPromotion() == 0) && !compStressCompile(STRESS_PHYSICAL_PROMOTION, 25))
```

Comment:
Should this be `||` instead or do we want to do physical promotion if `JitEnablePhysicalPromotion == 1` and `!compStressCompile(STRESS_PHYSICAL_PROMOTION, 25)` is true?

Reply:
It should be `&&` -- I want to allow it to be enabled by either `JitEnablePhysicalPromotion=1` or the stress mode. `compStressCompile` is always false in release builds